instrumentation/action_pack/test/test_helpers/app_config.rb

```diff
@@ -6,10 +6,22 @@
 
 class Application < Rails::Application; end
 require 'action_controller/railtie'
+require 'active_record'
+
+require 'active_record/railties/controller_runtime'
+ActiveSupport.on_load(:action_controller) do
```

Comment:
I know the Railtie is supposed to set up this on load hook but I couldn't figure out how to do that with this test application.

Tips appreciated, as I'd like to have as little unnecessary setup as possible. Without this the `db_runtime` value in the notification event is not present.

https://github.com/rails/rails/blob/fc71f585ba552d90b9b834a980e4f4688367dae9/activerecord/lib/active_record/railtie.rb#L284